--- /etc/postgresql/9.4/main/pg_hba.conf	2015-12-31 00:34:27.948948585 +0100
+++ /etc/postgresql/9.4/main/pg_hba.conf	2015-12-31 00:33:35.680603501 +0100
@@ -82,7 +82,7 @@
 # maintenance (custom daily cronjobs, replication, and similar tasks).
 #
 # Database administrative login by Unix domain socket
-local   all             postgres                                peer
+local   all             postgres                                trust

 # TYPE  DATABASE        USER            ADDRESS                 METHOD
